src/subsplit_dag_node.hpp

```diff
@@ -40,6 +40,9 @@ class SubsplitDAGNode {
   void AddLeafwardSorted(size_t node_id) { leafward_sorted_.push_back(node_id); }
   void AddRootwardRotated(size_t node_id) { rootward_rotated_.push_back(node_id); }
   void AddRootwardSorted(size_t node_id) { rootward_sorted_.push_back(node_id); }
+  const SizeVector &GetLeafwardOrRootward(bool leafward, bool rotated) const {
```

Comment:
Two arguments of the same type but different meaning always makes me a little queasy, but I can see how you are using this.

Reply:
That's why I was considering making enumerated types for these. Could help with readability?

Reply:
Sounds interesting, but I don't know if I totally get it. Enumerated types for leafward and rootward? Sounds like a good idea if we use a typesafe class enum.

Reply:
Yeah, leafward and rootward probably have less application.  But we are using boolean/size_t values enough for distinguishing sorted/rotated edges that enumeration could maybe make sense.

Reply:
OK, then let's mark it with a comment containing the issue number so it doesn't get forgotten.